lib/reactotron-core-client/src/serialize.ts

```diff
@@ -15,6 +15,21 @@ const INFINITY = "~~~ Infinity ~~~"
 const NEGATIVE_INFINITY = "~~~ -Infinity ~~~"
 // const NAN = '~~~ NaN ~~~'
 
+/**
+ * Fix BigInt serialization
+ * BigInts are not supported by JSON.stringify.  This is a workaround.
+ * https://github.com/GoogleChromeLabs/jsbi/issues/30#issuecomment-953187833
+ */
+declare global {
```

Comment:
// Extend BigInt.prototype.toJSON if BigInt is available and method is not already defined
if (typeof BigInt !== 'undefined' && !BigInt.prototype.toJSON) {
  BigInt.prototype.toJSON = function () {
    return this.toString() + "n"; // Append 'n' to distinguish BigInt
  };
}